packages/components/src/renderers/DefaultRenderer.tsx

```diff
@@ -48,6 +48,9 @@ export class DefaultRenderer extends React.Component<any, any> {
             }
         }
 
-        return <span>{display}</span>;
+        // issue 36941: when using the default renderer, add css so that line breaks as preserved
+        const cls = display?.indexOf('\n') > -1 ? 'whitespace-prewrap' : '';
+
+        return <span className={cls}>{display}</span>;
```

Comment:
Is there any harm in using the className even the value doesn't have a newline in it?  Seems less fragile (and slightly less work) to just always include the class.

Reply:
Good point. Nope, likely no harm in that. I'll pull in the changes from your PR to match the class name and implementation so their shouldn't be merge conflicts.